scripts/install.js

```diff
@@ -109,18 +110,63 @@ function checkAndDownloadBinary() {
       return;
     }
 
-    download(sass.getBinaryUrl(), sass.getBinaryPath(), function(err) {
-      if (err) {
-        console.error(err);
-        return;
-      }
+    var tmpPath = getTempPath();
+    if (!(sass.hasBinary(tmpPath))) { // download and install
+      download(sass.getBinaryUrl(), tmpPath, function(err) {
+        if (err) {
+          console.error(err);
+          return;
+        }
+        console.log('Binary downloaded at', tmpPath);
+        //copy only, if not already in vendor
+        if (tmpPath !== sass.getBinaryPath()) {
+          copyBinary(tmpPath);
+        }
+      });
 
-      console.log('Binary downloaded and installed at', sass.getBinaryPath());
-    });
+    } else { // install only
+      copyBinary(tmpPath);
+    }
   });
 }
 
 /**
+ * Find a temp folder for file
+ *
+ * @returns {string} temp folder including binary file name
+ * @api private
+ */
+
+function getTempPath() {
+  var candidateTmpDir = tmpdir() || process.env.npm_config_tmp;
```

Comment:
I saw the change and IMHO it's incorrect since `tmpdir()` can never return a falsy value. Also, even if there is a tmpdir there is no guarantee it can be written to. The previous behaviour was correct because it asserted that either `tmpdir()` or `process.env.npm_config_tmp` existed and could be written to.